projects/robots/universal_robots/resources/ros_package/ur_e_webots/README.md

```diff
@@ -15,21 +15,25 @@ roslaunch ur5_moveit_config ur5_moveit_planning_execution.launch
 For starting up RViz with a configuration including the MoveIt! Motion Planning plugin, run the following launch file (from the `universal_robot` ROS package):
 
 ```
-roslaunch ur5_moveit_config moveit_rviz.lnch config:=true
+roslaunch ur5_moveit_config moveit_rviz.launch config:=true
 ```
 
+## Multi Robots
+
+If your simulation use more than one robot using the `universal_robots_ros` controller, the `---node-name` controller arguments should be set to avoid name clashes between nodes.
```

Comment:
```suggestion
If your simulation uses more than one robot running the `universal_robots_ros` controller, the `---node-name` controller arguments should be set to avoid a name clash between nodes.
```